src/components/pageNav/index.jsx

```diff
@@ -8,7 +8,7 @@ import ArrowRight from '../site/icons/ArrowRight';
 import styles from './page-nav.module.scss';
 
 class PageNav extends React.PureComponent {
-  renderNavItem(type, item, baseUrl) {
+  renderNavItem(type, item, baseUrl, disableButton) {
```

Comment:
ok, so I think we can clean up some of this previous vs next button stuff.  First thing's first we'll provide a default value for `disableButton` so like: `renderNavItem(type, item, baseUrl, disableButton = false) {`.  That way, when we call this method for the previous button, we can make it so disableButton is always false...which should prove helpful

sidebar: this could also be affected by passing in `false` instead of `disableButton` when this method is called for the previous button, but I like the default args strat more for whatever reason